python/doc/examples/reliability_sensitivity/reliability/plot_form_explained.py

```diff
@@ -11,256 +11,307 @@
+# first marginal:
+#
+# .. math::
+#    :label: detT1
+#
+#    T_1 = \Phi^{-1} \circ F_1
 #
-ui = [distX1.computeCDF(xi[0]), xi[1]]
+transform_X1 = dist_X1.getIsoProbabilisticTransformation()
+inverse_transform_X1 = dist_X1.getInverseIsoProbabilisticTransformation()
 
 # %%
-# The second step is to build a standard unit gaussian from a uniform variable. This is done by a
-# simple call to the probit function. The point `zi` is said to be in the standard space.
-zi = [-ot.Normal().computeInverseSurvivalFunction(ui[0])[0], ui[1]]
+# We can implement the transformation using :math:`T_1` on the first components
+# directly using :math:`T` on both components `xi`:
+zi1D = [transform_X1([xi[0]])[0], xi[1]]
+zi2D = transformation(xi)
 
 # %%
-# The sought transform then maps a point in the physical space to the standard space :
-print(xi, "->", ui, "->", zi)
+# We can check the result of our experiment : we observe the results are the same.
+print("zi = ", zi)
+print("zi1D = ", zi1D)
+print("zi2D = ", zi2D)
 
 
 # %%
-# We also build the isoprobabilistic transform :math:`T_1` and its inverse :math:`T_1^{-1}` for the
-# first marginal :
-transformX1 = distX1.getIsoProbabilisticTransformation()
-inverseTransformX1 = distX1.getInverseIsoProbabilisticTransformation()
+# The model in the standard space is defined by:
+#
+# .. math::
+#
+#     \widetilde{\model} = \model \circ T^{-1}
+#
+# We can define it using the capacities of the composition of functions implemented in the library.
+g_tilde = ot.ComposedFunction(g, inverse_transformation)
 
 # %%
-# We can check the result of our experiment against :
+# The failure domain in the standard space is defined by:
 #
-# - the 2D-transform :math:`T` ;
-# - the 1D-transform :math:`T_1` and the second component unchanged ;
+# .. math::
+#
+#  \set{\widetilde{D}} = \{ (z_1, z_2) \in [0,+\infty[ \times \mathbb{R} \, | \,  \widetilde{\model}(z_1, z_2) \geq s \}
+#
+# and its boundary is defined by:
+#
+# .. math::
+#
+#    \partial \set{\widetilde{D}} = \{ (z_1, z_2) \in [0,+\infty[ \times \mathbb{R} \, | \,
+#       \widetilde{\model}(z_1, z_2) = s \}
 #
-# and observe the results are the same.
-zi1D = [transformX1([xi[0]])[0], xi[1]]
-zi2D = transformation(xi)
-print("zi = ", zi)
-print("zi1D = ", zi1D)
-print("zi2D = ", zi2D)
-
 
 # %%
-# We can represent the boundary of the event in the standard space : that is a composition of the
-# hyperbole :math:`h : x \mapsto 10/x` and the inverse transform :math:`T_1^{-1}` defined by
-# :math:`inverseTransformX1`.
-failureBoundaryPhysicalSpace = ot.SymbolicFunction(["x"], ["10.0 / x"])
-failureBoundaryStandardSpace = ot.ComposedFunction(
-    failureBoundaryPhysicalSpace, inverseTransformX1
-)
-x = np.linspace(1.1, 5.0, 100)
-cx = np.array([failureBoundaryStandardSpace([xi])[0] for xi in x])
+# We draw the graph of :math:`\widetilde{g}` in the standard space.
+graph_standard_space = g_tilde.draw([0.0, 0.0], [7.0, 7.0], [101] * 2)
```

Comment:
This is simpler than in the original code, indeed.

Reply:
yes...